### ISSUE TITLE
Redeem Xbox Purchases

### ISSUE THREAD

**Issue by beansszn** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#376]
### Issue Checklist

- [X] I have properly named the issue
- [X] I looked in issues/discussions, if my question was previously anwsered

### What is your question?

Hello,
From what I've read from previous issues, it is currently not known how to redeem Xbox purchases (through RedeemRealMoneyPurchases)? Has this changed since?

It would be great to no longer require Xbox Cloud Gaming (or a console) to claim items from Microsoft Redeem.

**Comment by Jaren8r** [comment]
Nope. As far as I'm aware, you would need an XSTS token with a title identity token, which would be impossible to get without authenticating to xbox live as a real xbox.
